test/test_macho.rb

```diff
@@ -28,262 +28,312 @@ def test_truncated_file
   end
 
   def test_load_commands
-    file = MachO::MachOFile.new(fixture(:x86_64, "hello.bin"))
-
-    file.load_commands.each do |lc|
-      assert lc
-      assert_equal MachO::LoadCommand, lc.class.superclass
-      assert_kind_of Fixnum, lc.offset
-      assert_kind_of Fixnum, lc.cmd
-      assert_kind_of Fixnum, lc.cmdsize
-      assert_kind_of String, lc.to_s
-      assert_kind_of Symbol, lc.type
-      assert_kind_of Symbol, lc.to_sym
+    files = [
+      MachO::MachOFile.new(fixture(:x86_64, "hello.bin")),
+      MachO::MachOFile.new(fixture(:ppc, "echo.bin"))
+    ]
```

Comment:
I'd prefer for this array to only contain the file names (`fixture(…)`) and the `MachOFile` instances be created inside the block of the following `each do`.